esmvaltool/preprocessor/_io.py

```diff
@@ -83,6 +83,8 @@ def concatenate(cubes):
 def _save_cubes(cubes, **args):
     """Save iris cube to file."""
     filename = args['target']
+    optimize_accesss = args['optimize_access']
+    del args['optimize_access']
```

Comment:
use `optimize_access = args.pop('optimize_access')` and remove the third s from `optimize_acesss` later on as well